TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/Robot.java

```diff
@@ -37,13 +42,22 @@
 import org.firstinspires.ftc.teamcode.api.hw.SmartMotor;
 import org.firstinspires.ftc.teamcode.api.hw.SmartServo;
 
-public class Robot extends AbstractRobot implements WheeledRobot, ArmRobot {
+import java.util.concurrent.Executor;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import lombok.Getter;
+
+public class Robot extends AbstractRobot implements WheeledRobot, StepWheeledRobot, ArmRobot, DistanceSensorRobot {
+    public static ExecutorService executorService;
+
     // Discuss if private is better idea
```

Comment:
if you're going to keep them public, it's a better idea to have them static. Else, use lombok getters and setters

Reply:
I have no idea what that's used for, so...

Reply:
below those lines sorry

Reply:
Fixed in 6bd254e